src/layers/normalise.jl

```diff
@@ -65,6 +65,27 @@ Specify `rng` to use a custom RNG instead of the default.
 Custom RNGs are only supported on the CPU.
 
 Does nothing to the input once [`Flux.testmode!`](@ref) is `true`.
+
+# Examples
+```jldoctest
+julia> m = Chain(Dense(1 => 1), Dropout(1));
+
+julia> Flux.trainmode!(m);
+
+julia> y = m([1]);
+
+julia> count(i->(i == 0), y) == m[2].p  # number of zeros == 1
```

Comment:
```suggestion
julia> y == [0]
```
Expresses more or less the same thing.